packages/block-library/src/navigation/edit/menu-inspector-controls.js

```diff
@@ -150,6 +152,27 @@ const MenuInspectorControls = ( props ) => {
 					>
 						{ __( 'Menu' ) }
 					</Heading>
+					<BlockControls group="inline">
+						<NavigationMenuSelector
```

Comment:
What do you think about only showing this control if there is more than 1 menu? That might limit any potential confusion for users who only have a single menu (i.e. fresh install one is created by fallback algorithm).

One counter argument is that having a toolbar option that shows/hides depending on context could be confusing to users of assistive tech. Continuity of tools is probably a major factor here.

Reply:
It also offers the import from classic option...